### PR TITLE
feat: add swapper/diversifier support to base

### PR DESCRIPTION
**Pull request by r0ohafza**

### CODE REVIEW

File: packages/splits-sdk/src/constants/index.ts [hunk 1]
```diff
@@ -17,12 +17,16 @@ const WATERFALL_MODULE_FACTORY_ADDRESS =
 const LIQUID_SPLIT_FACTORY_ADDRESS =
   '0xdEcd8B99b7F763e16141450DAa5EA414B7994831'
 const RECOUP_ADDRESS = '0xCbB386B801Ec72A5aB02AEB723dECd12f96EdE41'
-const PASS_THROUGH_WALLET_FACTORY_ADDRESS =
+const PASS_THROUGH_WALLET_FACTORY_ADDRESS_MAINNET =
   '0xF5aCC1568706Fbf9A55a77DdBe8DF907Da95dD6B'
+const PASS_THROUGH_WALLET_FACTORY_ADDRESS =
+  '0x52d6838957ec268cc5B50F17F9b490cbAb0A9E40'
 const SWAPPER_FACTORY_ADDRESS = '0xa244bbe019cf1BA177EE5A532250be2663Fb55cA'
 const UNI_V3_SWAP_ADDRESS = '0x981a6aC55c7D39f50666938CcD0df53D59797e87'
```
Comment (mihoward21) [inline packages/splits-sdk/src/constants/index.ts:25]: this is the same on base?

Comment (mihoward21) [reply packages/splits-sdk/src/constants/index.ts:25]: that's our integration contract

Comment (r0ohafza) [reply packages/splits-sdk/src/constants/index.ts:25]: yup

**Review by mihoward21 — APPROVED** [review]
just double checking that the intergration contract address is the same.

otherwise lgtm.

let me know if you want me to build and publish it, happy to do so.

**Pull request merged**